src/screens/Learn/CaseStudies/CaseStudiesLanding.tsx

```diff
@@ -60,15 +58,15 @@ const Landing: React.FC = () => {
         <MobileOnly>
           <TableOfContents items={getSectionItems(categories)} />
         </MobileOnly>
-        {categories.map((category: CaseStudyCategory) => {
+        {categoriesWithStudies.map((category: CaseStudyCategory) => {
```

Comment:
Consider exporting `categoriesWithSturies` with the studies already filtered, so the show/hide logic is fully contained in the `cms-content/learn/index.ts` file